apps/dg/components/game/game_controller.js

```diff
@@ -88,14 +88,39 @@ DG.GameController = DG.ComponentController.extend(
        * As of build ~0428 we allow data interactives to come to the front like any other component
        * @type {boolean}
        */
-      preventBringToFront: false,
+      preventBringToFront: function(key, value) {
+        var path = 'model.content.preventBringToFront';
+        if (value != null) {
+          this.setPath(path, value);
+        }
+        return this.getPath(path);
+      }.property(),
```

Comment:
Note: I tried using bindings for these, but could not get them to work bidirectionally.